Week7/REVIEW.md

```diff
@@ -1,9 +1,196 @@
 # REVIEW JavaScript week 7
 
+## Git Workflow
+
+To be provided
```

Comment:
here we can add a link to: https://github.com/HackYourFuture/Git/blob/master/Lecture-3.md

Reply:
the workflow is explained there